common/Services/IExperimentationService.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using DevHome.Common.Models;
+
+namespace DevHome.Common.Services;
+
+public interface IExperimentationService
+{
+    bool IsEnabled(string key);
+
+    void SetIsEnabled(string key, bool value);
+
+    List<ExperimentalFeature> Features { get; }

```

Comment:
Maybe this should be called `ExperimentalFeatures`, would be consistent with `AddExperimentalFeature` below.

Reply:
Fixed!